scripts/test-dist.js

```diff
@@ -4,7 +4,7 @@
 
 const path = require("path");
 const shell = require("shelljs");
-const tempy = require("tempy");
+const tmpDir = require("temp-dir");
```

Comment:
this is different - before you created a new directory inside of the OS temp dir - this just returns the path of existent temp dir without creating a new directory inside of it. So you run `npm init` the root temp dir, which is probably not what you want

Reply:
you are right, I didn't think through. I will revert that update when I back on desktop. thanks

Reply:
I have revert that change